src/plugin/cesium/sync/featureconverter.js

```diff
@@ -1653,6 +1654,11 @@ plugin.cesium.sync.FeatureConverter.prototype.olGeometryToCesium = function(feat
     geomType = 'ellipse';
   }
 
+  if (os.query.isWorldQuery(geometry)) {
```

Comment:
`os.query.isWorldQuery` doesn't work in 3857 because the tolerance of the `goog.math.nearlyEquals` call is too small given the projection units. We should change that function to test that the geometry's area is within a certain (very small) percent of the world area.

It might also be a good idea to cache the world area by projection, so we only have to call transformExtent/getArea on the world extent once per projection.